File: Cmdlets/UnregisterCultureAndRegionInfoCommand.vb
Imports System.Globalization
Imports System.Management.Automation

<Cmdlet("Unregister", "CultureAndRegionInfo")>
Public Class UnregisterCultureAndRegionInfoCommand
	Inherits Cmdlet

	<Parameter(Mandatory:=True)>
	Public Property Name() As String

	Protected Overrides Sub BeginProcessing()

		CultureAndRegionInfoBuilder.Unregister(Name)
	End Sub
End Class

File: WindowsCmdlets/Commands/UnregisterCultureAndRegionInfoCommand.cs
using System.Globalization;
using System.Management.Automation;

namespace TIKSN.Cmdlets.Commands
{
    [Cmdlet("Unregister", "CultureAndRegionInfo")]
    public class UnregisterCultureAndRegionInfoCommand : Cmdlet
    {
        [Parameter(Mandatory = true)]
        public string Name { get; set; }

        protected override void BeginProcessing()
        {
            CultureAndRegionInfoBuilder.Unregister(Name);
        }
    }
}